tpot/base.py

```diff
@@ -598,6 +614,7 @@ def _update_top_pipeline(self):
                 if pipeline_scores.wvalues[1] > top_score:
                     self._optimized_pipeline = pipeline
                     top_score = pipeline_scores.wvalues[1]
```

Comment:
`top_score` variable isn't needed any more if the same value is being stored in `self._optimized_pipeline_score`.